cmd/terraform/status.go

```diff
@@ -2,28 +2,16 @@ package main
 
 import (
 	"github.com/deislabs/porter-terraform/pkg/terraform"
-	"github.com/deislabs/porter/pkg/printer"
 	"github.com/spf13/cobra"
 )
 
 func buildStatusCommand(m *terraform.Mixin) *cobra.Command {
-	opts := struct {
-		rawFormat string
-		format    printer.Format
-	}{}
 	cmd := &cobra.Command{
 		Use:   "status",
 		Short: "Print the status of the terraform components in the bundle",
-		PreRunE: func(cmd *cobra.Command, args []string) error {
-			var err error
-			opts.format, err = printer.ParseFormat(opts.rawFormat)
-			return err
-		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return m.Status(printer.PrintOptions{Format: opts.format})
+			return m.Status()
 		},
 	}
-
-	cmd.Flags().StringVarP(&opts.rawFormat, "output", "o", "plaintext", "Output format. Allowed values are: plaintext, yaml, json")
```

Comment:
Looks like that command supports other output types, can we keep that?

https://www.terraform.io/docs/commands/show.html 

Maybe open an issue to add it back?

Reply:
Yea, if you are ok with it, I'll create an issue to track (requires a tf client bump as well).

Reply:
https://github.com/deislabs/porter-terraform/issues/5

Reply:
Yeah, you never need to twist my arm for a follow-up issue/PR. Just as long as it's called out, I trust ya. 😀